plugins/permission-common/src/PermissionClient.ts

```diff
@@ -91,35 +118,67 @@ export class PermissionClient implements PermissionAuthorizer {
   }
 
   /**
-   * Request authorization from the permission-backend for the given set of permissions.
+   * Request authorization from the permission-backend for the given set of
+   * permissions.
+   *
+   * @remarks
```

Comment:
Instead of duplicating method comments I suggest using {@inheritdoc PermissionEvaluator.authorize} and {@inheritdoc PermissionEvaluator.query}